tracer/src/Datadog.Trace/AppSec/Rcm/AsmProduct.cs

```diff
@@ -46,6 +48,12 @@ public void ProcessUpdates(ConfigurationStatus configurationStatus, List<RemoteC
 
             if (asmConfig.Actions != null)
             {
+                if (!actionsCleared)
```

Comment:
this and the following could now use a rebase, actions should clear automatically with removed files or empty arrays in fles like other objects